bigbluebutton-html5/imports/ui/components/audio/audio-test/container.jsx

```diff
@@ -1,24 +1,31 @@
-import React, { Component } from 'react';
+import React from 'react';
 import { createContainer } from 'meteor/react-meteor-data';
+import Service from '/imports/ui/components/audio/service';
```

Comment:
AudioService ?

Reply:
Only service in this container so `Service` will do

Reply:
I thought it was our convention.